doc/ipythonconsole.rst

```diff
@@ -51,10 +52,12 @@ To connect to an external kernel,
 #. Browse for or enter the path to the connection file from the previous step.
    If you're connecting to a local kernel, click :guilabel:`Ok` and Spyder should connect to the kernel; if a remote kernel, proceed to the final step.
 
+   As a convenience, kernel ID numbers (e.g. ``20345``) entered in the connection file path field will be expanded to :file:`{jupter_runtime_dir}/kernal-{id}.json` on your local machine.
```

Comment:
opps I wrote this. but it looks like `jupter_runtime_dir` is `runtime-dir` every where else.

I would go with `jupter-runtime-dir` everywhere to be more specific because when I first read `runtime-dir` I thought it was just the directory that I was in when I started the kernel, literally the directory at run time.

Reply:
Ugh, I should have noticed that inconsistency. Good point; I changed it to ``{jupyter/runtime/dir}`` to make it clearer its a path and not just a directory name. I also replaced the example PID with a more generic one (``1234``). which also matches the one in the dialog itself. Another PR incoming...